addons/binding/org.openhab.binding.yamahareceiver.test/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,30 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: YamahaReceiver Binding Tests
+Bundle-RequiredExecutionEnvironment: JavaSE-1.8
+Bundle-SymbolicName: org.openhab.binding.yamahareceiver.test;singleton:=true
+Bundle-Vendor: openHAB
+Bundle-Version: 2.4.0.qualifier
+Export-Package: org.openhab.binding.yamahareceiver;uses:="org.eclipse.smarthome.test"
+Fragment-Host: org.openhab.binding.yamahareceiver
+Import-Package: 
+ org.eclipse.jdt.annotation;resolution:=optional,
+ org.eclipse.smarthome.core.common.registry,
+ org.eclipse.smarthome.core.events,
+ org.eclipse.smarthome.core.thing,
+ org.eclipse.smarthome.core.thing.binding,
+ org.eclipse.smarthome.core.thing.binding.builder,
+ org.eclipse.smarthome.core.thing.util,
+ org.eclipse.smarthome.test.java,
+ org.eclipse.smarthome.test.storage,
+ org.hamcrest;core=split,
+ org.hamcrest.core,
+ org.junit;version="4.0.0",
+ org.mockito,
+ org.mockito.invocation,
+ org.mockito.stubbing,
+ org.osgi.framework,
```

Comment:
`org.openhab.binding.yamahareceiver` needs to be imported here.

Reply:
Could you apply some sorting here, like its done on most places?